### ISSUE TITLE
Unexpected message when running --reneg

### ISSUE THREAD

**Issue by svengo** [nabla-c0d3/sslyze#422]
**Describe the bug**
I get an error running `sslyze --reneg cloud.svengo.net`:
`nassl._nassl.OpenSSLError: error:140943F2:SSL routines:ssl3_read_bytes:sslv3 alert unexpected message`

**To Reproduce**
Use docker:

```
$ docker run -it --rm python:3.7 bash
root@44641328dc7d:/# pip install --upgrade setuptools
Collecting setuptools
  Downloading setuptools-46.1.3-py3-none-any.whl (582 kB)
     |████████████████████████████████| 582 kB 10.2 MB/s
Installing collected packages: setuptools
  Attempting uninstall: setuptools
    Found existing installation: setuptools 46.0.0
    Uninstalling setuptools-46.0.0:
      Successfully uninstalled setuptools-46.0.0
Successfully installed setuptools-46.1.3
root@44641328dc7d:/# pip install --upgrade sslyze
Collecting sslyze
  Downloading sslyze-3.0.0.tar.gz (1.0 MB)
     |████████████████████████████████| 1.0 MB 10.1 MB/s
Collecting nassl<3.2.0,>=3.0.0
  Downloading nassl-3.0.0-cp37-cp37m-manylinux2010_x86_64.whl (3.2 MB)
     |████████████████████████████████| 3.2 MB 19.8 MB/s
Collecting cryptography==2.5
  Downloading cryptography-2.5-cp34-abi3-manylinux1_x86_64.whl (2.4 MB)
     |████████████████████████████████| 2.4 MB 33.1 MB/s
Collecting tls-parser<1.3.0,>=1.2.0
  Downloading tls_parser-1.2.1.tar.gz (7.6 kB)
Collecting typing_extensions
  Downloading typing_extensions-3.7.4.1-py3-none-any.whl (20 kB)
Collecting asn1crypto>=0.21.0
  Downloading asn1crypto-1.3.0-py2.py3-none-any.whl (103 kB)
     |████████████████████████████████| 103 kB 36.2 MB/s
Collecting cffi!=1.11.3,>=1.8
  Downloading cffi-1.14.0-cp37-cp37m-manylinux1_x86_64.whl (400 kB)
     |████████████████████████████████| 400 kB 45.7 MB/s
Collecting six>=1.4.1
  Downloading six-1.14.0-py2.py3-none-any.whl (10 kB)
Collecting pycparser
  Downloading pycparser-2.20-py2.py3-none-any.whl (112 kB)
     |████████████████████████████████| 112 kB 37.2 MB/s
Building wheels for collected packages: sslyze, tls-parser
  Building wheel for sslyze (setup.py) ... done
  Created wheel for sslyze: filename=sslyze-3.0.0-py3-none-any.whl size=1028834 sha256=cf803b8a73fbe37e8e59ce5518116ecf8b641d5c1d88ab53a4449590b13f93e1
  Stored in directory: /root/.cache/pip/wheels/4b/7b/01/0ee51f3803ab8f505c5a78ffd04ae08ce4c0c5a2ad9d86a0de
  Building wheel for tls-parser (setup.py) ... done
  Created wheel for tls-parser: filename=tls_parser-1.2.1-py3-none-any.whl size=11382 sha256=ed0488a03cb504a5ee2b2258f4904267ee76c6d70a47a897e5a2180e9a25abae
  Stored in directory: /root/.cache/pip/wheels/c9/bb/02/5090d911be6f67099d22dcc3380fad4b5d747e66c72570ec99
Successfully built sslyze tls-parser
Installing collected packages: nassl, asn1crypto, pycparser, cffi, six, cryptography, tls-parser, typing-extensions, sslyze
Successfully installed asn1crypto-1.3.0 cffi-1.14.0 cryptography-2.5 nassl-3.0.0 pycparser-2.20 six-1.14.0 sslyze-3.0.0 tls-parser-1.2.1 typing-extensions-3.7.4.1
root@44641328dc7d:/# python -m sslyze --reneg cloud.svengo.net

 CHECKING HOST(S) AVAILABILITY
 -----------------------------

   cloud.svengo.net:443                       => 194.59.204.142




 SCAN RESULTS FOR CLOUD.SVENGO.NET:443 - 194.59.204.142
 ------------------------------------------------------

 * Error when running --reneg:
       You can open an issue at https://github.com/nabla-c0d3/sslyze/issues with the following information:

       * Server: cloud.svengo.net:443 - 194.59.204.142
       * Scan command: session_renegotiation

       Traceback (most recent call last):
         File "/usr/local/lib/python3.7/site-packages/sslyze/scanner.py", line 224, in get_results
    server_info, server_and_scan_cmd_to_completed_futures[server_and_scan_cmd]
         File "/usr/local/lib/python3.7/site-packages/sslyze/plugins/session_renegotiation_plugin.py", line 100, in result_for_completed_scan_jobs
    result_enum, value = job.result()
         File "/usr/local/lib/python3.7/concurrent/futures/_base.py", line 428, in result
    return self.__get_result()
         File "/usr/local/lib/python3.7/concurrent/futures/_base.py", line 384, in __get_result
    raise self._exception
         File "/usr/local/lib/python3.7/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
         File "/usr/local/lib/python3.7/site-packages/sslyze/plugins/session_renegotiation_plugin.py", line 148, in _test_client_renegotiation
    ssl_connection.ssl_client.do_renegotiate()
         File "/usr/local/lib/python3.7/site-packages/nassl/legacy_ssl_client.py", line 81, in do_renegotiate
    self.do_handshake()
         File "/usr/local/lib/python3.7/site-packages/nassl/ssl_client.py", line 180, in do_handshake
    self._ssl.do_handshake()
       nassl._nassl.OpenSSLError: error:140943F2:SSL routines:ssl3_read_bytes:sslv3 alert unexpected message



 SCAN COMPLETED IN 0.05 S
 ------------------------
root@44641328dc7d:/# exit
```

**Expected behavior**
cloud.svengo.net runs traefik 2.1.19 with some TLS tweaks. sslyze should generate a report and no error :-)

**Python environment (please complete the following information):**
 - OS: docker 19.03.8 on Ubuntu 18.04.4 LTS
 - Python version: 3.7

**Comment by nabla-c0d3** [comment]
Thanks for the detailed report

**Comment by nabla-c0d3** [comment]
Fixed in 3.0.1.
